zellij-utils/assets/config/default.yaml

```diff
@@ -392,3 +392,10 @@ plugins:
 #   - true (default)
 #   - false
 #mouse_mode: false
+
+# Configure the scroll back buffer size
+# This is the number of lines zellij stores for each pane in the scroll back
+# buffer. Excess number of lines are discarded in a FIFO fashion.
+# Valid values: positive integers
+# Default value: 10_000
```

Comment:
Our `yaml` parser seems to have problems with this way of specifying integers, it seems to work well with `10000` for me.